File: Vishwas RNO 40/Java/Assign3/src/InhousePatient.java
package hospital;

public class InhousePatient extends Patient {

	private double discount;

	public InhousePatient(int id, String na, int bt, int d, double dis) {
	super(id, na, bt,d);
	discount = dis;
	}

	public double getBillAmount() {
		double amount = super.getBillAmount();{
		discount = amount >= 5000 ? 5.0 : 0;
		}
		return amount-amount * discount /100;
	}
}

File: Vishwas RNO 40/DotNet/Assignment 2/DemoApp/hospital/InHousePatient.cs
namespace hospital;

class InHousePatient : Patient
{
    public double Discount {get; set;}

    public InHousePatient(int PatientId, string PatientName, int BedType, int NoOfdays, double Discount)
    {
        base.PatientId = PatientId;
        base.PatientName = PatientName;
        base.BedType = BedType;
        base.NoOfdays = NoOfdays;
        this.Discount = Discount;        
    }

    public override double GetBillAmount() {
		
        double amount = base.GetBillAmount();
		
        Discount = amount >= 5000 ? 5.0 : 0;
		
		return amount-amount * Discount /100;
	}
}